### PR TITLE
Refactored header to make it easier to swap the nav

### PR DESCRIPTION
**Pull request by vbusch**
### Type of change

- Refactoring

### Description

Refactored the header, to allow a new one to easily be swapped in.

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: console/console-init/ui/src/AppLayout.tsx [hunk 1]
```diff
@@ -17,6 +18,7 @@ import {
   RootModal
 } from "components";
 import { AppRoutes } from "Routes";
+import { AppNavHeader } from "./AppNavHeader";
```
Comment (anukritijha) [inline console/console-init/ui/src/AppLayout.tsx:21]: AppNavHeader file is unavailable.

Comment (vbusch) [reply console/console-init/ui/src/AppLayout.tsx:21]: Added missing file.  Thank you.

File: console/console-init/ui/src/AppNavHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
```
Comment (k-wall) [inline console/console-init/ui/src/AppNavHeader.tsx:1]:
```suggestion
/*
 * Copyright 2020, EnMasse authors.
 * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
 */
 
import * as React from "react";
```

Comment (k-wall) [reply console/console-init/ui/src/AppNavHeader.tsx:1]: Missing license.

Comment (vbusch) [reply console/console-init/ui/src/AppNavHeader.tsx:6]: updated

**Comment by vbusch** [comment]
> Looks reasonable to me. Isn't the equivalent change needed on master and release-0.33?

Yes.  Created this PR on master, and was planning to cherry-pick to release-0.33 and release-0.32.2.  Should those be listed on the milestone as well?

**Pull request merged**